CMake/HHVMExtensionConfig.cmake

```diff
@@ -951,6 +951,13 @@ function (HHVM_EXTENSION_INTERNAL_HANDLE_LIBRARY_DEPENDENCY extensionID dependen
       HHVM_EXTENSION_INTERNAL_ADD_LINK_LIBRARIES(${MYSQL_CLIENT_LIBS})
     endif()
   elseif (${libraryName} STREQUAL "pgsql")
+    FIND_PACKAGE(PostgreSQL)
+    if (${PostgreSQL_VERSION_STRING} VERSION_LESS "9.1.0")
+      message(STATUS "System PostgreSQL version is too old: Disabling extension.")
+      set(HHVM_EXTENSION_${extensionID}_ENABLED_STATE 2 CACHE INTERNAL "" FORCE)
```

Comment:
Why are you doing this rather than `HHVM_EXTENSION_INTERNAL_SET_FAILED_DEPENDENCY(${extensionID} ${dependencyName})`?

This should also be completely replacing the FIND_PATH stuff below.

I'll put up a diff in a moment to switch over to that.